### PR TITLE
feat(UIS-9) Get all inventory items

### PR DESCRIPTION
**Pull request by StaceyJin**
JIRA: https://champlainsaintlambert.atlassian.net/browse/UIS-9

Get all inventory items functionality was implemented to BACKEND ONLY:

- method added to InventoryItemService and InventoryItemServiceImpl;
- controller with method added;
- relevant unit tests added.

### CODE REVIEW

File: src/test/java/com/um/inventoryservice/DataLayer/InventoryItemRepositoryTest.java [hunk 1]
```diff
@@ -31,6 +35,32 @@ public void insertInventoryItem (){
...
+        Flux<InventoryItem> find = inventoryItemRepository.findAll();
+        Publisher<InventoryItem> composite = Mono
+                .from(setup)
+                .thenMany(find);
+
+        StepVerifier
+                .create(composite)
+                .consumeNextWith(foundInvItem ->{
+                    assertEquals(invItem.getInventoryItemId(), foundInvItem.getInventoryItemId());
+//                    assertEquals(invItem.getStockItemDTO(), foundInvItem.getStockItemDTO());
```
Comment (Kam1st) [inline src/test/java/com/um/inventoryservice/DataLayer/InventoryItemRepositoryTest.java:58]: do we need this commented line?

File: src/test/java/com/um/inventoryservice/PresentationLayer/InventoryControllerIntegrationTest.java [hunk 2]
```diff
@@ -51,10 +51,34 @@ void insertInventoryItem() {
...
+        webTestClient
+                .get()
+                .uri("/inventory")
+                .accept(MediaType.APPLICATION_JSON)
+                .exchange()
+                .expectStatus().isOk()
+                .expectHeader().contentType(MediaType.APPLICATION_JSON)
+                .expectBody()
+                .jsonPath("$[0].inventoryItemId").isEqualTo(inventoryItem.getInventoryItemId())
+//                .jsonPath("$[0].stockItemDTO").isEqualTo(inventoryItem.getStockItemDTO())
```
Comment (Kam1st) [inline src/test/java/com/um/inventoryservice/PresentationLayer/InventoryControllerIntegrationTest.java:76]: do we need this part of the test?

**Review by Kam1st — APPROVED** [review]

File: src/test/java/com/um/inventoryservice/DataLayer/InventoryItemRepositoryTest.java (hunk 1 above)
Comment (StaceyJin) [reply src/test/java/com/um/inventoryservice/DataLayer/InventoryItemRepositoryTest.java:58]: The reference address for both StockItems will always be different, therefore the test will always fail with that assertEquals statement. The line will be removed.

File: src/test/java/com/um/inventoryservice/PresentationLayer/InventoryControllerIntegrationTest.java (hunk 2 above)
Comment (StaceyJin) [reply src/test/java/com/um/inventoryservice/PresentationLayer/InventoryControllerIntegrationTest.java:76]: Please refer to response to Repository Test comment.

**Pull request merged**